app/src/main/java/io/intrepid/skeleton/base/BaseMvpActivity.java

```diff
@@ -0,0 +1,50 @@
+package io.intrepid.skeleton.base;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.CallSuper;
+import android.support.annotation.NonNull;
+
+public abstract class BaseMvpActivity<T extends BasePresenter> extends BaseActivity implements BaseView {
+
+    protected T presenter;
+
+    @NonNull
+    public abstract T createPresenter(PresenterConfiguration configuration);
+
+    protected abstract int getLayoutResourceId();
+
+    @CallSuper
+    protected void onViewCreated(Bundle savedInstanceState) {
+        PresenterConfiguration configuration = getSkeletonApplication().getPresenterConfiguration();
+        presenter = createPresenter(configuration);
+        presenter.onViewCreated();
+    }
+
+    @Override
+    @CallSuper
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        presenter.bindView(this);
```

Comment:
Why is there a `bindView` call here in the `onActivityResult`?

Wouldn't it be sufficient to have a `bindView` call during the `onCreate` (through `onViewCreated`), and then call a `start` or `stop` method (or something similar) in the Activity's `onStart' and 'onStop' ?